cfgmgr/vrfmgr.cpp

```diff
@@ -207,6 +225,46 @@ void VrfMgr::doTask(Consumer &consumer)
         auto vrfName = kfvKey(t);
 
         string op = kfvOp(t);
+		// Mgmt VRF table event handling for in-band management
+        if (consumer.getTableName() == CFG_MGMT_VRF_CONFIG_TABLE_NAME) {
```

Comment:
{ -> next line. Please do it for rest of the changes

Reply:
done